File: Source/ses_io/src/wrappers/fortran/wrap_ses_read_word.f90


#undef DEBUG_WRAP

SES_ERROR_FLAG function ses_read_word_f90(the_handle, pt_buffer)

   implicit none

   SES_FILE_HANDLE, INTENT(IN) :: the_handle
   SES_WORD_REFERENCE  :: pt_buffer

   SES_ERROR_FLAG :: return_value, ses_read_word

#ifdef DEBUG_WRAP
   print *, "ses_read_word_f90:  the_handle is ", the_handle
#endif

   return_value = ses_read_word(the_handle, pt_buffer)

#ifdef DEBUG_WRAP
   print *, "ses_read_word_f90:  return_value is ", return_value
   print *, "ses_read_word_f90:  the_buffer is ", pt_buffer
#endif

   ses_read_word_f90 = return_value

end function ses_read_word_f90

File: Source/ses_io/src/wrappers/fortran/wrap_ses_read_word.c


#include "ses_defines.h"
#include "ses_globals.h"
#include "ses_internals.h"
#include "ses_externs.h"

#include "stdio.h"
#include <stdlib.h>

#include <stdio.h>

#undef DEBUG_WRAP

#ifdef LC_UNDER
ses_error_flag ses_read_word_(ses_file_handle* pt_handle, ses_word_reference buffer) {
#else

#ifdef UC_UNDER
ses_error_flag SES_READ_word_(ses_file_handle* pt_handle, ses_word_reference buffer) {
#else

#ifdef LC_NOUNDER
ses_error_flag ses_read_word(ses_file_handle* pt_handle, ses_word_reference buffer) {
#else

#ifdef UC_NOUNDER
ses_error_flag SES_READ_word(ses_file_handle* pt_handle, ses_word_reference buffer) {
#endif

#endif
#endif
#endif

  ses_error_flag return_value = SES_NO_ERROR;

  ses_file_handle the_handle = *pt_handle;
#ifdef DEBUG_WRAP
  printf("wrap_ses_read_word.c:  the_handle is %d\n", the_handle);
#endif

  return_value = ses_read_word(the_handle, buffer);
#ifdef DEBUG_WRAP 
  printf("wrap_ses_read_word.c:  return_value is %d\n", return_value);
  printf("wrap_ses_read_word.c:  buffer is %d\n", *buffer);
#endif
  return return_value;

}

 
